docs/README.md

```diff
@@ -41,117 +41,74 @@ In the product theme template, add the `condition-layout.{context}` block, repla
+| `subject`   | `string`  | Defines, according to the product context where the block in declared in, which data is needed from the UI to validate the value chosen in the `object` prop. Check below the possible values for this prop. | `undefined`   |
+| `arguments` | `object`  | Defines the condition parameters. Notice: this prop value varies according to the value set to the `subject` prop. Check below the table for the `subject`'s possible values and their expected arguments. | `undefined`   |
+| `toBe`      | `boolean` | Whether the data fetched in the `subject` prop must met the predefined conditions to render the new layout (`true`) or not (`false`). | `true` |
 
-- Possible `subject` prop's values provided by the product context:
+Possible values for the `subject` prop:
 
-| Subject | Type | Description |
-| ----------------- | ------- | ------------------------------------- |
-| `productId` | `value` | Product's IDs displayed on the UI. |
-| `categoryId` | `value` | Category's IDs displayed on the UI. |
-| `brandId` | `value` | Brand's IDs displayed on the UI. |
-| `selectedItemId` | `value` | ID of the item being selected by the user on the UI. |
-| `productClusters` | `array` | List of product clusters on the UI. |
-| `categoryTree` | `array` | List of categories on the UI. |
-| `specificationProperties` | `array` | List of product specifications. |
-| `areAllVariationsSelected` | `value` | Whether all product variations available on the page were selected by the user (`true`) or not (`false`). | 
-
-:information_source: *Since the Condition Layout can only be used with product contexts, only the subjects listed above are needed for the proper functioning of the `condition` block. Remember to choose the subject's value according to the value passed to the `object` prop*.
+| Subject                    | Description            | Arguments      |
+| -------------------------- | ---------------------- | -------------- |
+| `productId`                | Product's IDs currently displayed on the UI.  | `{ id: string }` |
+| `categoryId`               | Category's IDs currently displayed on the UI. | `{ id: string }` |
+| `brandId`                  | Brand's IDs currently displayed on the UI.    | `{ id: string }` |
+| `selectedItemId`           | ID of the item currently selected by the user.   | `{ id: string }` |
+| `productClusters`          | List of product clusters currently displayed on the UI.   | `{ id: string }` |
+| `categoryTree`             | List of categories currentluy displayed on the UI.   | `{ id: string }`  |
+| `specificationProperties`  | List of product specifications currently displayed on the UI. | `{ name: string, value: string }`. Notice: `value` is an optional prop. If omitted, only the specification name (`name`) will be checked. |
+| `areAllVariationsSelected` | Whether all product variations currently available on the UI were selected by the user (`true`) or not (`false`). | No arguments are expected. |
```

Comment:
No, because it doesn't accept any arguments. Imagine it as a function:

`areAllVariationsSelected()` would return `true` or `false`. One can define if it wants a true/false value by using the `toBe` prop!